src/js-marketing-notification.ts

```diff
@@ -1,32 +1,31 @@
 import { LocalStorageManager } from './github/gtmsportswear/js-local-storage-manager@1.0.2/local-storage-manager';
 
 export interface Notification {
-  tabContent: string,
-  heading: string;
-  buttonTitle: string;
-  image?: string;
-  content?: string;
+  tabContent: Element,
```

Comment:
For future reference, we'll have to alter this interface again I think since "tabContent" really isn't valid for more than one block.